docs/Getting Started.md

```diff
@@ -44,37 +44,71 @@ Next, make sure everything is set up correctly with a quick test run.
 
 ## Run your First Simulation Example: The Dam Break with Reef3D
 
-In this example, you wish to evaluate the potential impact of a dam break scenario. 
-Your goal is to simulate the dam break using the open-source hydrodynamics [Reef3d simulator](https://github.com/REEF3D) to understand the dynamics of the water flow, predict the areas 
-that would be affected, and propose effective mitigation strategies.
+In this example, you will use the open-source hydrodynamics REEF3D simulator to simulate a [dam break scenario](https://github.com/REEF3D/REEF3D/tree/master/Tutorials/REEF3D_CFD/9_1%202D%20Dam%20Break).
 
-Running this simulation requires substantial computational resources and precise 
-configuration, challenges that have previously slowed down your project.
+With this first example, you learn that running simulations via Inductiva API is
+not much different from running them in your local machine. That's the magic
+of it all! 
 
-With Inductiva, you have the opportunity to run the dam break simulation with 
-your existing configuration files.
```

Comment:
In this first example, you'll discover the simplicity of running simulations with the Inductiva API – it allows you to run the dam break simulation with your existing configuration files just like you would on your local machine, except it's on the Cloud. 

what do you think? (merging lines 51 till 55)

Reply:
I think this was deprecated!